apps/web-giddh/src/app/daybook/advance-search/daybook-advance-search.component.html

```diff
@@ -327,7 +341,7 @@ <h3 class="modal-title bg" id="modal-title">{{ commonLocaleData?.app_advance_sea
                     </div>
                 </ng-container>
                 <div class="col-xs-12 col-sm-12 text-right">
-                    <button class="btn btn-success" type="button" (click)="go()">
+                    <button mat-raised-button color="primary" type="button" (click)="go()">
```

Comment:
@ravindersinghdang please remove this color from html and copy the button class name properly.

Reply:
Done